src/app/core/services/vault/vault.service.ts

```diff
@@ -230,6 +235,80 @@ export class VaultService {
     this.localStorageStore.putState(newState);
   }
 
+  importFactomAddress(type: FactomAddressType, nickname: string, publicAddress: string, privateAddress: string, vaultPassword: string) {
+    return defer(async () => {
+      try {
+        const state = this.localStorageStore.getState();
+        const decryptedVault = await encryptor.decrypt(vaultPassword, state.vault);
+
+        decryptedVault[publicAddress] = privateAddress;
```

Comment:
I am confused here: why do we store the `privateAdress` as a value for `publicAddress` key?

Reply:
Ah, never mind, it's obviously so that you can find the corresponding private address to a given public address more easily.